tensorly/decomposition/_tr.py

```diff
@@ -268,6 +270,260 @@ def tensor_ring_als(
     return tr_decomp
 
 
+def compute_lev_score_dist(matrix):
```

Comment:
I would rename it just `leverage_score_dist` - the compute is implied and it's clearer than lev? Can we also add a reference? Would it be useful outside of TR? If so we could move it to `tensorly.metrics`

Reply:
@JeanKossaifi Thank you for the suggestions! I updated the name and added a reference. This could be useful outside of TR (e.g., for sampled CP decomposition), so I moved the function to `tensorly.metrics`.